src/java_bytecode/java_bytecode_language.h

```diff
@@ -100,8 +101,8 @@ class java_bytecode_languaget:public languaget
                                     //  - checkcast / instanceof
                                     //  - array bounds check
                                     //  - array size for newarray
-  size_t max_nondet_array_length;   // maximal length for non-det array creation
-  size_t max_user_array_length;     // max size for user code created arrays
+  unsigned max_nondet_array_length; // maximal length for non-det array creation
+  unsigned max_user_array_length;   // max size for user code created arrays
   lazy_methodst lazy_methods;
```

Comment:
This was changed so that the call to `optionst::set_option` would not be ambiguous.